client/src/components/pages/admin/DataVisualizationPage.js

```diff
@@ -0,0 +1,373 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import Chart1 from './LineChart';
+import ProgressBar from 'react-bootstrap/lib/Progressbar';
+import Chart2 from './DonutChart';
+
+import { getUsers, getAlerts} from 'actions';
+
+require('./tableStyles.scss');
+
+
+class DataVisualizationPage extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.defaultState = {
+            width: 0,
+            
+        }
+
+        this.state = this.defaultState;
+    }
+
+    componentWillMount() {
+        this.props.getUsers();
+        this.props.getAlerts();
+    }
+
+    filterAdvocates(users) {
+        let advocates = users.filter(function(user) { return user.role == "advocate"});
+        return advocates;
+    }
+
+    filterAdmin(users){
+        let admin = users.filter(function(user) { return user.role == "admin"});
+        return admin;
+    }
+
+    filterProviders(users){
+        let providers = users.filter(function(user) { return user.role == "providers"});
+        return providers;
+    }
+
+    alertAmountPrevious(alerts){
+        console.log(alerts[0])
+        var currentTime = new Date();
+
+    }
+
+    render() {
+        let {users, current_user, alerts} = this.props;
+        let advocates = this.filterAdvocates(users);
+        let providers = this.filterProviders(users);
+        let admin = this.filterAdmin(users);
+
+        let advocatesAmount = advocates.length;
+        let providersAmount = providers.length;
+        let alertAmount = alert.length;
+        let alertAmountPrevious = this.alertAmountPrevious(alerts);
+
+        var dashboard;
+        dashboard = (
+            <div className="dashboard">
+                <div className="row tile_count">
+                    <div className="col-md-2 col-sm-4 col-xs-6 tile_stats_count">
+                      <span className="count_top"><i className="fa fa-user"></i> Total Advocates</span>
+                      <div className="count">{advocatesAmount}</div>
+                      <span className="count_bottom"><i className="green">4% </i> From last Week</span>
+                    </div>
```

Comment:
There is a lot of repeated code here. This is what making individual components is for. 

Please look into [stateless components](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions) for a simple approach to reusable components that don't need to do much other than render some chunk of HTML.